benchopt/cli/main.py

```diff
@@ -36,20 +36,20 @@
               help="Include <solver_name> in the benchmark. By default, all "
               "solvers are included. When `-s` is used, only listed solvers"
               " are included. To include multiple solvers, "
-              "use multiple `-s` options.", autocompletion=get_solvers)
+              "use multiple `-s` options.", shell_complete=get_solvers)
 @click.option('--force-solver', '-f', 'forced_solvers',
               metavar="<solver_name>", multiple=True, type=str,
               help="Force the re-run for <solver_name>. This "
               "avoids caching effect when adding a solver. "
               "To select multiple solvers, use multiple `-f` options.",
-              autocompletion=get_solvers)
+              shell_complete=get_solvers)
```

Comment:
Through this PR I realize that there may be some autocompletion working? But hitting tab after having typed a solver name does not do anything for me, e.g. in this situation:

`(base) ➜  lasso git:(main) ✗ benchopt run . -s sk`

Reply:
You need to enable the autocompletion following the first paragraph of this page:
https://benchopt.github.io/cli.html

Maybe a useful feature would be to have these instruction display by running `benchopt config autocomplete`?